Command/InitAclCommand.php

```diff
@@ -0,0 +1,124 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\AclBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Security\Acl\Dbal\Schema;
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\Schema\SchemaException;
+
+/**
+ * Installs the tables required by the ACL system.
+ *
+ * @author Johannes M. Schmitt <schmittjoh@gmail.com>
+ *
+ * @final since version 3.4
+ */
+class InitAclCommand extends ContainerAwareCommand
+{
+    private $connection;
+    private $schema;
+
+    /**
+     * @param Connection $connection
+     * @param Schema     $schema
+     */
+    public function __construct($connection = null, Schema $schema = null)
+    {
+        if (!$connection instanceof Connection) {
```

Comment:
You're right, the message is a bit off-context in case of null passed by default. Then again this only impacts people using the _core_ commands outside the bundle ecosystem. SF itself doesnt trigger it. So im not sure it's really worth the effort, yet this PR can improve it for aclbundle of course :)